lib/lib-storage/src/Upload.ts

```diff
@@ -427,6 +438,31 @@ export class Upload extends EventEmitter {
     });
   }
 
+  private __calculatePartSize(targetPartSizeBytes: number, minPartSize: number): number {
+    const calculatedPartSize = Math.floor(targetPartSizeBytes / this.MAX_PARTS);
+    return Math.max(minPartSize, calculatedPartSize);
+  }
+
+  private __validateUploadPart(dataPart: RawDataPart, partSize: number): void {
```

Comment:
why is `partSize` an argument to this method when it is only ever the same instance variable `this.partSize`?